templates/helpers.yaml

```diff
@@ -31,6 +31,7 @@ spec:
           command: ["ipa-helper"]
           args: ["--identity", "{{ trimPrefix "h" .Release.Name }}",
             "--port", {{ .Values.mpcPort | quote }},
+            "--log-file", "/var/log/ipa.log",
```

Comment:
do we have enough space in `/var` to keep the logs?

Reply:
yes, that's were we kept the ones with 8gb for the 1B run we just did